addons/web_editor/static/lib/odoo-editor/src/powerbox/Powerbox.js

```diff
@@ -131,7 +131,7 @@ export class Powerbox {
                     ev => {
                         ev.preventDefault();
                         ev.stopImmediatePropagation();
-                        this._currentValidate();
+                        this._currentValidate(command);
```

Comment:
This part belongs in the next commit. It was confusing for me to read the other commit, seeing you added an argument to the method but not where you use it.

Reply:
done